tracpro/polls/views.py

```diff
@@ -543,8 +544,8 @@ def lookup_field_label(self, context, field, default=None):
         def lookup_field_value(self, context, obj, field):
             if field == 'region':
                 return obj.contact.region
-            elif field == 'group':
-                return obj.contact.group
+            elif field == 'groups':
+                return ', '.join(group.name.encode('utf-8') for group in obj.contact.groups.all())
```

Comment:
We might want to order these groups by name the same as we do a few lines farther down, just for consistency, but not a big deal either way.